source/common/json/json_loader.cc

```diff
@@ -2,254 +2,475 @@
 
 #include <cstdint>
 #include <fstream>
+#include <limits>
+#include <sstream>
+#include <stack>
 #include <string>
 #include <vector>
 
+#include "common/common/utility.h"
+#include "common/filesystem/filesystem_impl.h"
+
 #include "spdlog/spdlog.h"
 
-// Do not let RapidJson leak outside of this file.
+// Do not let RapidJson leak outside of json_loader.
```

Comment:
The old comment is correct, but to make it more clear, you can change it to json_loader.cc